src/controller/python/chip/ChipCoreBluetoothMgr.py

```diff
@@ -538,88 +520,16 @@ def bgScanStop(self):
         self.logger.info("scanning stopped")
 
     def connect(self, identifier):
-        """ API to initiate BLE connection to peripheral device whose identifier == identifier."""
-        self.logger.info("trying to connect to " + identifier)
-
-        if self.connect_state:
-            self.logger.error("ERROR: Connection to a BLE device already exists!")
-        else:
-            for peripheral in self.peripheral_adv_list:
-                p = peripheral.peripheral
-                devIdInfo = peripheral.getPeripheralDevIdInfo()
-                if not devIdInfo:
-                    # Not a chip device
-                    continue
-                p_id = str(p.identifier().UUIDString())
-                p_name = str(p.name())
-
-                self.logger.debug(p_id + " vs " + str(identifier))
-                self.logger.debug(p_name + " vs " + str(identifier))
-
-                if p_id == str(identifier) or p_name == str(identifier) or str(devIdInfo.discriminator) == str(identifier):
-                    self.loop_condition = False
-                    self.peripheral = p
-                    self.manager.connectPeripheral_options_(p, None)
-
-                    self.runLoopUntil(LoopCondition("connect", 15.0))
-                    # Cleanup when the connect fails due to timeout,
-                    # otherwise CoreBluetooth will continue to try to connect after this
-                    # API exits.
-                    if not self.connect_state:
-                        self.manager.cancelPeripheralConnection_(p)
-                        self.peripheral = None
-
-                    break
-
-        ret = True if self.loop_condition and self.connect_state else False
-
-        resString = "connect " + ("success" if ret else "fail")
-        self.logger.info(resString)
-
-        return ret
+        """ No longer supports ble connect in chip device controller """
```

Comment:
Consider instead:

```py
""" DEPRECATED. Connect not supported and will be removed."""
```

and a GH TODO isse to remove (or explain what the plan is for the future) 

Same for the entries below.

if not supported, why are we not removing outright?

Reply:
Done, and created 4548 for deeper clean up.